README.md

```diff
@@ -1,16 +1,16 @@
-# Odin project sign up form
+# Odin dashboard project
 
-> One paragraph statement about the project.
+> The project contains some information .
 
 ![screenshot](./app_screenshot.png)
 
-Additional description of the project and its features.
+The project is one of the project that tells about the doer .
 
 ## Built With
 
-- Major languages,
-- frameworks,
-- technologies used
+- HTML,
+- CSS,
```

Comment:
And also here you need to state what you use in html, what you use in css, not everything but enough to describe what they are all about